security/secret-protection/docs/secret-protection.rst

```diff
@@ -0,0 +1,281 @@
+
+.. _secret-protection-tutorial:
+
+Tuturial: Secret Protection
+===========================
+
+Overview
+--------
+
+Confluent’s **Secret Protection** feature encrypts secrets in configuration
```

Comment:
I don't think `Secret Protection` should be bold (here or in the line below)?

Reply:
You're right. Will update